usagov/usagov-2021
# USAGOV-1828-Analytics-Dashboard-Dropdown-And-Link: Updates to Dropdow…

…n and H1


## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1828

## Description

Added selected attribute to appropriate dropdown value on each page. 
Moved h1 text into link tag. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps
- Easiest way to test this is using "Go Live" extension in vscode. 
- From website-analytics/index.html use Go Live to open preview site. 
- No data will be loaded that is fine for this ticket. 
- First make sure the H1 text behaves like a link 
- Second on the drop down chose another value and select Go. 
- Make sure that value stays selected and does not revert to first value as it previously did (USA.gov and USA.gov/es/). 
- From one of these pages if you select the H1 text it should take you to the "USA.gov and USA.gov/es/" page

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
